config.schema.json

```diff
@@ -13,7 +13,8 @@
     "perp_pool_banners",
     "cross_selling_source_tokens",
     "external_chain_channels",
-    "additional_ibc_token_config"
+    "additional_ibc_token_config",
+    "demex_trading_league_config"
```

Comment:
Not all of the networks are hosting Demex Trading League competitions at the same time, hence can make `demex_trading_league_config` optional